release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2933,6 +2933,46 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.42
+[id="ocp-4-17-43_{context}"]
+=== RHBA-2025:19314 - {product-title} {product-version}.43 bug fix update and security
+
+Issued: 05 November 2025
+
+{product-title} release {product-version}.43 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2025:18235[RHBA-2025:19314] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:19312[RHBA-2025:19312] advisory.
```

Comment:
This first link didn't get updated. It says the new ID, but the actual link is the old ID